core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1,5 +1,33 @@
 package org.infinispan.util.logging;
 
+import static org.jboss.logging.Logger.Level.DEBUG;
```

Comment:
ROFL! You put the static imports at the top, and @slaskawi  puts them at the bottom in his PRs, can we stop this madness plz? :)